common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -979,4 +980,15 @@ public static void invalidatePluginCaches() {
   public static List<String> parseCmdParameters(Optional<String> cmdArgsLine) {
     return cmdArgsLine.map(ParametersListUtil::parse).orElse(ContainerUtil.newArrayList());
   }
+
+  public static void invokeLater(Runnable task) {
```

Comment:
Maybe let's rename it if it's not always "later"

Reply:
Renamed to `invokeLaterIfNeeded` to maintain compatibility with the underlying idea method